cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,10 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName    = "sa"
+	ServiceAccountPublicKeyName  = "sa.pub"
```

Comment:
@luxas I know you mentioned creating `sa.crt` and `sa.key`. How do you feel about `sa.pub` to make it clear that it is the public portion of a key?